AI/pipelines/daily_routine.py

```
@@ -0,0 +1,202 @@
+project_root = os.path.abspath(os.path.join(current_dir, "../.."))
+if project_root not in sys.path:
+    sys.path.append(project_root)
+
+# 모듈 Import
+from AI.modules.signal.workflows.inference import run_inference
+from AI.modules.trader.policy import decide_order
+from AI.modules.analysis.generator import ReportGenerator
+from AI.modules.collector.market_data import update_market_data
+from AI.libs.database.repository import save_executions_to_db, save_reports_to_db
+from AI.modules.signal.core.data_loader import SignalDataLoader
+from AI.libs.database.repository import save_executions_to_db, save_reports_to_db, get_current_position 
```

Ruff (0.14.11)

Redefinition of unused `save_executions_to_db` from line 25: `save_executions_to_db` redefined here

Remove definition: `save_executions_to_db`

(F811)